code/mccSearch.py

```diff
@@ -421,25 +374,25 @@ def find_cloud_elements(mergImgs, timelist, mainStrDir, lat, lon, TRMMdirName=No
                 currFrameCEs.append(cloudElementDict)
 
```

Comment:
line 308 should be 
TRMMArea = TRMMnumOfBoxes * userVariables.XRES * userVariables.YRES